src/librbd/cache/rwl/LogEntry.cc

```diff
@@ -46,6 +50,12 @@ std::ostream &operator<<(std::ostream &os,
   return entry.format(os);
 }
 
+bool GenericWriteLogEntry::is_unsequenced_and_sp_entry_uncompleted() {
```

Comment:
Perhaps negate it and rename it to something more intuitive like:
```suggestion
bool GenericWriteLogEntry::is_persisted() {
```

Reply:
@dillaman Done. use can_flush() instead.